src/components/mutually-exclusive/mutually-exclusive.date.spec.js

```diff
@@ -127,7 +155,93 @@ describe('script: mutually-exclusive', () => {
 
       describe('when the user clicks the mutually exclusive option', () => {
         beforeEach(async () => {
-          await page.click('#date-exclusive-exclusive-option');
+          await page.click('#date-exclusive-option');
+        });
+
+        it('then the aria alert shouldnt say anything', async () => {
+          await page.waitForTimeout(SCREEN_READER_TIMEOUT_DELAY);
+
+          const alertText = await page.$eval('.ons-js-exclusive-alert', (node) => node.textContent);
+          expect(alertText).toBe('');
+        });
+      });
+    });
+  });
+  describe('date-year', () => {
+    beforeEach(async () => {
+      await setTestPage('/test', renderComponent('date-input', EXAMPLE_MUTUALLY_EXCLUSIVE_DATE_SINGLE_PARAMS));
+    });
+
+    describe('Given the user populated the date input', () => {
+      beforeEach(async () => {
+        await page.type('#date-mutually-exclusive-year', '2018');
+      });
+
+      describe('when the user clicks the mutually exclusive option', () => {
+        beforeEach(async () => {
+          await page.click('#date-exclusive-option');
+        });
+
+        it('then the mutually exclusive option should be checked', async () => {
+          const isChecked = await page.$eval('#date-exclusive-option', (node) => node.checked);
+          expect(isChecked).toBe(true);
+        });
+
+        it('then the date input should be cleared', async () => {
+          const yearValue = await page.$eval('#date-mutually-exclusive-year', (node) => node.value);
+          expect(yearValue).toBe('');
+        });
+
+        it('then the aria alert should tell the user that the date input has been cleared', async () => {
+          await page.waitForTimeout(SCREEN_READER_TIMEOUT_DELAY);
+
+          const alertText = await page.$eval('.ons-js-exclusive-alert', (node) => node.textContent);
+          expect(alertText).toBe('What year was your last MOT? cleared.');
+        });
+      });
+    });
+
+    describe('Given the user has checked the mutually exclusive exclusive option', () => {
+      beforeEach(async () => {
+        await page.click('#date-exclusive-option');
+      });
+
+      describe('when the user populates the dateInput', () => {
+        beforeEach(async () => {
+          await page.type('#date-mutually-exclusive-year', '2018');
+        });
+
+        it('then the mutually exclusive option should be checked', async () => {
```

Comment:
then the mutually exclusive option should 'not' be checked instead

Reply:
I am happy to approve after you make the small change requested

Reply:
Done, thought I'd go for "unchecked" instead because thats whats been done in other tests